backends-velox/src/test/scala/org/apache/spark/sql/execution/VeloxParquetWriteForHiveSuite.scala

```diff
@@ -118,20 +128,33 @@ class VeloxParquetWriteForHiveSuite extends GlutenQueryTest with SQLTestUtils {
         "CREATE TABLE t (c int, d long, e long)" +
           " STORED AS PARQUET partitioned by (c, d)")
       withSQLConf("spark.sql.hive.convertMetastoreParquet" -> "true") {
-        checkNativeStaticPartitionWrite(
-          "INSERT OVERWRITE TABLE t partition(c=1, d)" +
-            " SELECT 3 as e, 2 as e",
-          native = false)
+        if (SparkShimLoader.getSparkVersion.startsWith("3.4")) {
+          checkNativeStaticPartitionWrite(
+            "INSERT OVERWRITE TABLE t partition(c=1, d)" +
+              " SELECT 3 as e, 2 as e",
+            native = false)
+        } else {
+          checkNativeStaticPartitionWrite(
+            "INSERT OVERWRITE TABLE t partition(c=1, d)" +
+              " SELECT 3 as e, 2 as e",
+            native = false)
```

Comment:
Should be native = true

Reply:
@ayushi-agarwal Velox backend doesn't support dynamic partition write.